teammapper-backend/src/map/types.ts

```diff
@@ -46,6 +46,7 @@ export interface IMmpClientNodeBasics {
 export interface IMmpClientNode extends IMmpClientNodeBasics {
   coordinates: IMmpClientCoordinates
   detached: boolean
+  hidden: boolean
```

Comment:
not needed anymore, if we only do this on the client side?

Reply:
Would have to check, but if IMmpClientNode is only used for client/server comms then it's good to remove. I obviously need the hidden value on client side to actually hide nodes, but I think we primarily use UserNodeProperties for that.